### PR TITLE
responsive navbar and basic stylesheet added

### PR DESCRIPTION
**Pull request by rosapalli**

### CODE REVIEW

**Comment by keikomorimoto** [comment]
Great design, thanks Rosa. I have approved the pull request 👍 

The outstanding tasks you / Kim mentioned will be added to the trello broad so somebody else can pick it up as well. 
1. logo is behind the nav bar texts and need to be moved. 
2. Hide nav bar when scrolled down
3. Change the location of search bar. 

Re 1, here is my suggestions ;
You could move the logo outside of  <div class="navbar-header">
= move line 21-22 to line 12-13 on Header.php 
https://www.w3schools.com/bootstrap4/bootstrap_navbar.asp

or depending how you want to display your logo, this site may be useful. 
https://codepen.io/bootstrapped/pen/KwYGwq

File: models/post.php [hunk 1]
```diff
@@ -121,7 +121,7 @@ public static function uploadFile(string $title) {
         }
 
         $tempFile = $_FILES[self::InputKey]['tmp_name'];
-        $path = "C:/xampp/htdocs/blog/views/images/";
```
Comment (keikomorimoto) [inline models/post.php:124]: Old path location needs to be updated to "blog"

**Review by keikomorimoto — APPROVED** [review]
Please see my comment.

**Comment by rosapalli** [comment]
Hi Keiko,

Thanks for looking at it. The logo should not get behind the navbar text
now in any screen size. I thought I had resolved this.

Re scrolling down, I removed the blog title from the navbar so that it
disappears when you scroll down.

The only outstanding issue should be that the search bar is not fully
aligned to the right.

I'll try to look into it after we've adapted the search function to MVC.

Rosa

**Pull request merged**